action-server/actions/action_suspect_mild_symptoms_exposure_extra_recommendations.py

```diff
@@ -0,0 +1,21 @@
+from typing import Any, Dict, List, Text
+
+from rasa_sdk import Action, Tracker
+from rasa_sdk.executor import CollectingDispatcher
+
+
+class ActionSuspectMildSymptomsExposureExtraRecommendations(Action):
+    def name(self) -> Text:
+        return "action_suspect_mild_symptoms_exposure_extra_recommendations"
+
+    def run(
+        self,
+        dispatcher: CollectingDispatcher,
+        tracker: Tracker,
+        domain: Dict[Text, Any],
+    ) -> List[Dict[Text, Any]]:
+
+        dispatcher.utter_template("utter_remind_stay_home_self_isolate", tracker)
```

Comment:
I didn't know before but I got a warning lately: in future versions of rasa `utter_template` will be deprecated. We should prefer `dispatcher.utter_message(template=x)`. Obviously the same comment applies to all the occurrences in this PR.